analyze/analyses/dominators/emit.rs

```diff
@@ -34,11 +35,7 @@ impl traits::Emit for DominatorTree {
         ) {
             assert_eq!(id == items.meta_root(), depth == 0);
 
-            if *row == opts.max_rows() {
-                return;
-            }
-
-            if depth > opts.max_depth() {
+            if *row > opts.max_rows() || depth > opts.max_depth() {
```

Comment:
minor point, but I combined the two early return conditions, since neither is especially dense and it makes this local function a little vertically smaller

Reply:
& as mentioned, the `>` seems to have fixed a separate issue. maybe sometimes code motion _is_ helpful :P